### PR TITLE
[BLOG] Add whats-next-for-pydatasparse

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-whats-next-for-pydatasparse-quansight.vercel.app/blog/whats-next-for-pydatasparse).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. 
## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/whats-next-for-pydatasparse.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+published: March 11, 2020
+description: ''
+author: hameer-abbasi
+category: [PyData Ecosystem]
+featuredImage:
+  src: /posts/whats-next-for-pydatasparse/blog_feature_var1.svg
+  alt: 'An illustration of a brown and a dark brown hand coming towards each other to pass a business card with the logo of Quansight Labs.'
+hero:
+  imageSrc: /posts/whats-next-for-pydatasparse/blog_hero_var1.svg
+  imageAlt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.
```
Comment (MarsBarLee) [inline apps/labs/posts/whats-next-for-pydatasparse.md:12]:
```suggestion
  imageAlt: 'An illustration of a brown hand holding up a microphone, with some graphical elements highlighting the top of the microphone.'
```

File: apps/labs/posts/whats-next-for-pydatasparse.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: 'Planned architectural work for PyData/Sparse'
+published: March 11, 2020
+description: ''
+author: hameer-abbasi
+category: [PyData Ecosystem]
```
Comment (noatamir) [inline apps/labs/posts/whats-next-for-pydatasparse.md:6]:
```suggestion
category: [PyData ecosystem]
```

File: apps/labs/posts/whats-next-for-pydatasparse.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+title: 'Planned architectural work for PyData/Sparse'
+published: March 11, 2020
+description: ''
```
Comment (noatamir) [inline apps/labs/posts/whats-next-for-pydatasparse.md:4]:
```suggestion
description: 'An update about recent research activities and upcoming changes'
```

**Review by noatamir — APPROVED** [review]
LGTM. merging

**Pull request merged**